mslib/msui/wms_control.py

```diff
@@ -613,6 +652,21 @@ def display_progress_dialog(self):
         self.pdlg.setModal(True)
         self.pdlg.show()
 
+    def display_capabilities_dialog(self):
+        logging.debug("showing capabilities dialog")
+        self.cpdlg.reset()
+        self.cpdlg.setValue(1)
+        self.cpdlg.setModal(True)
+        self.cpdlg.show()
+
+    def stop_capabilities_retrieval(self):
+        logging.debug("stopping capabilities retrieval")
+        try:
+            self.capabilities_worker.quit()
+            self.capabilities_worker.disconnect()
+        except TypeError:
+            pass
```

Comment:
is it worth to add a logging here?

Reply:
The TypeError just means that there are no signals connected to the worker.
Basically I expected that case to be outright ignored but QT still throws an exception for it, which is completely irrelevant.

Reply:
ts ts ts QT